### ISSUE TITLE
Use LLVM Integrated Tester for test files

### ISSUE THREAD

**Issue by jmah** [jmah/Clang-KeyPathValidator#1]
http://llvm.org/docs/CommandGuide/lit.html
http://llvm.org/docs/TestingGuide.html

**Comment by AlexDenisov** [comment]
I've missed this part in my original post (http://railsware.com/blog/2014/02/28/creation-and-using-clang-plugin-with-xcode/), but can support you, if you don't mind ;-)

**Comment by jmah** [comment]
That would be great! I haven't spent the time to figure out the config to run tests in my own directory. Have you done so?

**Comment by AlexDenisov** [comment]
Yep, I have a skeleton for this stuff. I'm going to sent you a PR soon.
